ansible/deploy.yml

```diff
@@ -35,18 +38,42 @@
 
     project_has_composer: no
 
-    # project_post_build_commands:
-    # - trigger a restart for gunicorn
-
     project_finalize: true
 
   roles:
     - f500.project_deploy
 
-  post_tasks:
-    - name: install pipenv libraries
-      command: /usr/local/bin/pipenv install
-    # copy nginx setting
-    # copy supervisor
-    # reload supervisor
-    # run gunicorn
+  tasks:
+    - name: install deps
+      shell: "pipenv install"
+      args:
+        chdir: "{{ project_root }}/current"
+    - name: Template a file to /etc/files.conf
+      template:
+        src: dotenv.j2
+        dest: "{{ project_root }}/current/.env"
+        owner: deploy
+        group: deploy
+        mode: "0755"
+    - name: collect static files for django
+      shell: "pipenv run ./manage.py collectstatic"
```

Comment:
Out of curiosity, where is it set exactly which python version is being run?

Reply:
I believe this is taken care of by the Pipfile, if you list the the version of python in it.

Which we do:

https://github.com/thegreenwebfoundation/greenwebfoundation-admin/blob/master/Pipfile#L33

We have pyenv on the box as well

Reply:
You mentioned that it by mistake ran python 2 once, how did that happen?

Reply:
I was totally wrong about that. I thought some error I was seeing was because of a change in libraries from py2 to py3, meant that a library was missing that would normally be in py3.

It turned out I hadn't run the pipenv install, and that I should have gone to bed earlier the night before…